packages/material-ui/src/RadioGroup/RadioGroup.js

```diff
@@ -11,21 +11,21 @@ class RadioGroup extends React.Component {
 
   constructor(props) {
     super();
-    this.isControlled = props.value != null;
+    this.wasControlledOnInit = props.value != null;
```

Comment:
Since we don't intend to switch we might as well keep the name. This implies the behavior changes during the lifetime of the component.